Azure/sonic-buildimage
# [202012][BRCM TH3] Add SOC properties to prevent FDB events during warmboot

#### Why I did it
This is to fix issue reported by (https://github.com/Azure/sonic-buildimage/issues/8558)
The fix requires 2 changes:

1.  in BRCM SAI 4.3.5.2 which can be found in this PR (https://github.com/Azure/sonic-buildimage/pull/9754)
2.  This PR itself where it needs to set 2 SOC properties in order to activate the code changed in BRCM SAI 4.3.5.2

#### How to verify it

1. Repeat the steps documented in issue (https://github.com/Azure/sonic-buildimage/issues/8558) without these 2 SOC properties on a TH3 based HWSKU (Z9332F) and with any BRCM SAI (including 4.3.5.2) and see the issue.
2. Enable these two SOC properties and repeat the same steps again with Z9332F running with BRCM SAI 4.3.5.2 and observe that the problem no longer can be reproduced.

#### Which release branch to backport (provide reason below if selected)
Once verified that the fix that went into SAI 4.3.5.2 is also present in the SAI 6.0 branch, we should then port this to master branch (running BRCM SAI 6.0.x).
There is no plan to port this to 



#### Description for the changelog
[202012][BRCM TH3] Add SOC properties to prevent FDB events during warmboot


#### A picture of a cute animal (not mandatory but encouraged)
